src/pages/careers/platform-engineer.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { createUseStyles } from 'react-jss';
+import { renderToString } from 'react-dom/server';
+
+import { SEO, SitewideFooter, LayoutControl, SitewideHeader } from 'components';
+import Hero from 'components/careers/Hero';
+import Footer from 'components/careers/Footer';
+import Main from 'components/careers/Main';
+import { backstageLink } from 'components/careers/links';
+
+const useStyles = createUseStyles(() => ({
+  sitewideHeaderWrapper: {
+    marginBottom: 40,
+    paddingLeft: 16,
+    paddingRight: 16,
+  },
+}));
+
+const ROLE_NAME = 'Platform Engineer';
+const TYPEFORM_SLUG = 'platform-engineer';
+const HEADLINE = 'Help companies adopt backstage.io';
```

Comment:
Personally, this headline makes me think of solutions engineer roles. I would point to the platform being built instead I think.

Reply:
remedied to "Take an active role in shaping the future of our platform"